great_expectations/expectations/row_conditions.py

```diff
@@ -56,7 +57,7 @@ def _set_notnull(s, l, t) -> None:  # noqa: E741 # ambiguous name `l`
 ops = (gt ^ lt ^ ge ^ le ^ eq ^ ne).setResultsName("op")
 fnumber = Regex(r"[+-]?\d+(?:\.\d*)?(?:[eE][+-]?\d+)?").setResultsName("fnumber")
 punctuation_without_apostrophe = punctuation.replace('"', "").replace("'", "")
-condition_value_chars = alphanums + punctuation_without_apostrophe
+condition_value_chars = alphanums + punctuation_without_apostrophe + SPACE_CHARS
```

Comment:
adds `SPACE_CHARS` to valid `condition_value_chars`, which is turned into the condition value